lib/player.js

```diff
@@ -1865,6 +1952,11 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
           const now = Date.now() / 1000;
           const delta = now - startTime;
           this.stats_.setDrmTime(delta);
+          // Currently LCEVC is not supported with DRM Streams,
+          // hence hiding the canvas.
```

Comment:
Where does this restriction come from?

Is LCEVC metadata inside the mdat box with the base media?  Or is it in a separate box somewhere?

DRM should only encrypt the media data inside the container, never the mp4 itself.  If the metadata is outside mdat, I would expect that you'd be able to access it even for DRM content.

Or does LCEVC require you to read some pixels from the base media into the canvas instead of just overlaying the canvas with some areas left transparent?  I know you are not generally able to read pixels from DRM videos into a canvas.

Whatever the explanation, it would be nice to add it to this comment.

Reply:
Added a comment regarding this.